test/sprints.e2e-spec.ts

```diff
@@ -228,27 +287,62 @@ describe("Sprints Controller (e2e)", () => {
         });
 
         it("should return 404 if meetingId is invalid", async () => {
+            const { access_token, refresh_token } = await loginAndGetTokens(
+                "jessica.williamson@gmail.com",
+                "password",
+                app,
+            );
             const meetingId = 9999;
             return request(app.getHttpServer())
                 .get(`/voyages/sprints/meetings/${meetingId}`)
-                .set("Cookie", accessToken)
+                .set("Cookie", [access_token, refresh_token])
```

Comment:
no change required here but just curious, is refresh_token needed? I wonder why it's added now